lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -395,6 +427,7 @@ def filter(tag, time, record)
           log.error("output #{time} #{tag} #{record}")
         end
       end
+      transform_event(record)
```

Comment:
move this up between `check_for_match_and_format` and `add_pipeline_metadata`

Also - this will be run against every tag.  I think you should add a `event_trans_tag` parameter which takes the tag(s) to match, as was done for the formatters and elasticsearch_index_names.

Reply:
One of the transformations I wanted to do was:
```
pipeline_metadata.collector.original_raw_message <- message
```
and `add_pipeline_metadata()` create the field `pipeline_metadata.collector`, wouldn't we want to do the event transformation after calling `add_pipeline_metadata()` or should I implement this logic in `add_pipeline_metadata()`?

Similar with this transformation
```
@dest_time_name                                  <- event.metadata.creationTimestamp
```
if this happens before (the original timestamp transformation)[https://github.com/wozniakjan/fluent-plugin-viaq_data_model/blob/97f327dc55dd1a00652cf0271d01d345331c5624/lib/fluent/plugin/filter_viaq_data_model.rb#L426-L429], it will get lost. Or would you recommend not moving the creationTimestamp?

Reply:
Ah, I see.  Then change `add_pipeline_metadata` as you need - add a parameter for `original_raw_message`, etc.